spec/factories/news_items.rb

```diff
@@ -1,6 +1,8 @@
 FactoryGirl.define do
+
   factory :news_item do
-    
+    title  'my-title'
+    published_at Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.